### ISSUE TITLE
Allow a non-executable to be set as the default command

### ISSUE THREAD

**Issue by mojavelinux** [tj/commander.js#742]
When designing a CLI that uses command functions, it should still be possible to set a command as the default command if no command is specified. This is pretty much the same functionality as the existing default command, except it will be used for non-executable commands.

```js
cli
  .command('build', { isDefault: true })
  .description('Run the build')
  .action(() => {
    console.log('build!')
  })
```

**Comment by mojavelinux** [comment]
I'd be fine with saying that the two types of defaults are mutually exclusive if that turns out to be a challenge. Chances are, you are either designing one type of CLI or another.

**Comment by shadowspawn** [comment]
Support for setting a non-executable as the default command is included in the published pre-release of Commander v5.0.0 (#1163)

**Comment by shadowspawn** [comment]
Commander v5.0.0 has been released.

https://github.com/tj/commander.js/releases/tag/v5.0.0
